.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="cypress" />
+import {
+  deleteDashboard,
+  deleteAPIKey,
+  clickDeleteButton,
+  verifySuccesOfAction
+} from './utils'
+
+it('allows user to log out', () => {
+  cy.login();
+  cy.visit('/logout');
+  cy.get('.login-content-box').should('exist');
+})
+
+it('allows creating a dashboard with a panel', () => {
+  const CLOSE_DIALOGUE_BUTTON = 'button[aria-label="Close dialogue"]';
+
+  cy.login();
+  cy.visit('dashboard/new');
+  cy.get('button').contains('Add a new panel')
+    .should('be.visible').click();
+  cy.get('[value="Panel Title"]').clear().type(Cypress.env('panel_title'));
+  cy.get('textarea#description-text-area')
+    .should('exist').type(Cypress.env('panel_description'));
+  cy.get('button').contains('Apply')
+    .should('be.visible').click();
+  cy.get('.panel-title').contains(Cypress.env('panel_title'));
+  cy.get('[aria-label="Save dashboard"]').click();
+  cy.get(CLOSE_DIALOGUE_BUTTON)
+    .should('be.visible');
+  cy.get('[name="title"]').clear().type(Cypress.env('dashboard_title'));
+  cy.get('[type="submit"]')
+    .should('be.visible').click();
+  verifySuccesOfAction();
+  deleteDashboard();
+})
+
+it('checks if it is possible to upload a dashboard as JSON file', () => {
+  const IMPORT_DASHBOARD_BUTTON = '[data-testid="data-testid-import-dashboard-submit"]';
+  
+  cy.login();
+  cy.visit("dashboard/import");
+  cy.get('label').contains('Upload JSON').click();
+  cy.get('input[type=file]').selectFile('cypress/fixtures/test-dashboard.json',
+  {force:true});
+  cy.get(IMPORT_DASHBOARD_BUTTON).click();
+  cy.visit("dashboards");
+  cy.get('div').contains('New test dashboard')
+    .should('exist').click();
+  deleteDashboard();
+})
+
+it('allows inviting a user', () => {
+  cy.login();
+  cy.visit('/org/users/invite');
+  cy.get('[name="loginOrEmail"]').clear().type(Cypress.env('test_email_address'));
+  cy.get('[name="name"]').clear().type(Cypress.env('invite_user_name'));
```

Comment:
I usually also check in this kind of command that the field actually has the expected text (`.should('have.value',XXXX);`. Do you think it is worth it?

Reply:
I read the Cypress documentation and I see this makes sense, especially because of [this Cypress feature](git push origin HEAD). I'll do it!

Reply:
Now that we are using dynamic data, we cannot assert expecting the exact value, since it will change all the time. This can remain as a good practice whenever feasible.